code/modules/mob/living/carbon/xenomorph/abilities/ability_helper_procs.dm

```diff
@@ -40,7 +40,9 @@
 				to_chat(src, SPAN_WARNING("This [O.name] is too tough to be melted by your weak acid."))
 				return
 
-		if(O.density || istype(O, /obj/structure))
+		if(istype(O, /obj/item/hardpoint))
+			wait_time = 100 //you are not supposed to be able to easily combat-melt irreplaceable things.
```

Comment:
this should be a check on the atom level, probably a proc

Reply:
Made it into [separate PR](https://github.com/cmss13-devs/cmss13/pull/771).